exec/context_test.go

```diff
@@ -70,3 +71,14 @@ func Test_Context_Populate(t *testing.T) {
 	_, moreItems := <-testContextChannel
 	a.False(moreItems)
 }
+
+func Test_RandStringBytes(t *testing.T) {
+	a := assert.New(t)
+
+	correlationId1 := randStringBytes(10)
+	correlationId2 := randStringBytes(10)
```

Comment:
Once in hundred years, correlationId1 and correlationId2 might be the same. Maybe test two randStringBytes of different length?